src/main/webapp/utilities/MapsInterface.js

```diff
@@ -0,0 +1,105 @@
+// TODO(tzavidas): seperate the interface from the logic, by using an implementation for easier testing (like the bridge pattern)
+class MapInterface {
+    /**
+     * @param element the DOM element the map to be rendered
+     * @param centerCoordinates coordinates object that will be the center of the map
+     * @param options (optional) additional parameters that might be needed
+     * @param zoom (on options) initial zoom of the map
+     * @param directionsService (on options) mock object for directionsService, can be used for testing
+     * @param directionsRenderer (on options) mock object for directionsRenderer, can be used for testing
+     */
```

Comment:
@maxwbuckley is this acceptable way of describing what each property on the options object is doing? Maybe there is another way to do this?

Reply:
I added the way I think it is done above.